### PR TITLE
Assingnment 1 - Task 2.

### PR DESCRIPTION
**Pull request by Adil-Jussupov**
# Updated titles and descriptions on the Onboarding views

## :recycle: Current situation & Problem
The main goal of this PR is to adapt the text/symbols of the onboarding page of the template app to PICS app.

## :gear: Release Notes 

- Within the onboarding flow of the application (the views that pop up after first installing and opening up the application), adjusted the text components as well as the symbols to reflect the use case of our application.
- Adjusted the first onboarding view to give an introduction to our application. Picked appropriate symbols and text components that fit the use case of our application.
- Repeated this for the second onboarding view.

## :books: Documentation
The reason for this PR is to provide user quick intro to the app, that describes only main features in order to decrease the stress of first usage of the app. 

## :white_check_mark: Testing
no testings were conducted.

## :pencil: Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md):
- [x] I agree to follow the [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: PICS/Onboarding/Welcome.swift [hunk 1]
```diff
@@ -21,23 +21,23 @@ struct Welcome: View {
             areas: [
                 OnboardingInformationView.Content(
                     icon: {
-                        Image(systemName: "apps.iphone")
+                        Image(systemName: "chart.line.uptrend.xyaxis")
```
Comment (philippzagar) [inline PICS/Onboarding/Welcome.swift:24]: Great use of SF symbols! 🚀

File: PICS/Resources/Localizable.xcstrings [hunk 2]
```diff
@@ -180,7 +180,7 @@
         "en" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "The onboarding module allows you to build an onboarding flow like this one."
+            "value" : "Introduce yourself once (ID, name, gender identity, date of birth)"
```
Comment (philippzagar) [inline PICS/Resources/Localizable.xcstrings:205]: Nice to see that you're using String catalogs!

File: PICS/Resources/Localizable.xcstrings [hunk 3]
```diff
@@ -190,7 +190,7 @@
         "en" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Onboarding"
+            "value" : "Sigh in"
```
Comment (philippzagar) [inline PICS/Resources/Localizable.xcstrings]: Please fix the typo

File: PICS/Resources/Localizable.xcstrings [hunk 4]
```diff
@@ -210,7 +210,7 @@
         "en" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "HL7 FHIR"
+            "value" : "Fill the quastinary "
```
Comment (philippzagar) [inline PICS/Resources/Localizable.xcstrings]: Please fix the typo

File: PICS/Resources/Localizable.xcstrings [hunk 5]
```diff
@@ -524,7 +524,7 @@
         "en" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "This application demonstrates several Spezi features & modules."
+            "value" : "This application helps to:"
```
Comment (philippzagar) [inline PICS/Resources/Localizable.xcstrings]: Please try to come up with a better suited subtitle that fits the content of the rest of the view

**Review by philippzagar — CHANGES_REQUESTED** [review]
Hi @Adil-Jussupov ,

Thank you for the PR and the improvements. It's a great first version that only needs some more very minor changes!

Please use the PR template that is in the Pull Request description. Also, fill out the Documentation and Testing part of the PR template, it's completely fine to state that you didn't do anything in that aspect within your PR.

Please re-request a review once you have updated the PR description and addressed some of the in-line comments and I am happy to approve the PR to get it merged 🚀

**Comment by philippzagar** [comment]
@Adil-Jussupov Ensure that you incorporate and re-request feedback until the start of class today!

**Comment by philippzagar** [comment]
@Adil-Jussupov Is this PR ready to review? Ensure that you properly fill in the Pull Request description and update the branch with the `main` branch. Also, ensure to incorporate all in-line feedback from me

File: PICS/Resources/Localizable.xcstrings (hunk 5 above)
Comment (Adil-Jussupov) [reply PICS/Resources/Localizable.xcstrings]: Tried to use better wording

**Comment by Adil-Jussupov** [comment]
> @Adil-Jussupov Is this PR ready to review? Ensure that you properly fill in the Pull Request description and update the branch with the `main` branch. Also, ensure to incorporate all in-line feedback from me

@philippzagar  Hi Pillip. Thank you for your comments. I updated the code and resubmitted it for additional review.

**Review by philippzagar — APPROVED** [review]
Thanks for incorporating the feedback, looks good now, feel free to merge the PR! 🚀 
I updated your branch with the `main` branch so that you can merge the feature branch directly, but please keep that in mind for future PRs

**Pull request merged**